### DISCUSSION TITLE
How does global request limit work?

### DISCUSSION THREAD

**Discussion by ggogel** [seerr-team/seerr#583] [Q&A] [answered]
I'm wondering how the settings "global movie request limit" and "global series request limit" work. Is this a total number of requests a user can ever make, or is there a cycle (weekly, monthly, yearly) when these "tokens" are refreshed? Also, is it possible that users can still file requests after their limit is exceeded? Will they be blocked or do they just need to get manually approved?

**Comment by fallenbagel** [comment, marked as answer]
https://docs.overseerr.dev/using-overseerr/settings#global-movie-request-limit-and-global-series-request-limit

It's a configurable limit. After they reach this limit they cannot request until it refreshes

**Reply by ggogel** [reply]
Thanks, I just tried it and it seems like you can actually configure the cycle between 1 and 100 days.

I think it would be great if there is an option to allow users to make additional requests but only accept them as soon as they have additional "tokens" or if an admin manually approves it. This would be more like an approval limit rather than a request limit.

**Comment by j007bond007** [comment]
Hello, I'm attempting to move from Ombi and there's a few concerning things that I thought would already be in Seerr that aren't (don't get me wrong - very greatful for this free app!):

- "Global" Movie/Series Request isn't clear - global to me sounds like cumulatively across all users, everyone can only hit x number of movies/series requests (so if user A and user B request 10 - and the global limit is 20 - they nobody can request anymore). Or is it that it's per-user and a "global default"?
- How can we define more series limits? e.g. The Simpsons has 37 seasons - so if that's counted as 1 series, that's a large risk allowing that through. Can we define number of episodes or seasons for series?
- Is it possible to use group-based limits? E.g. maybe when the OIDC feature is implemented you can pull the in the users groups, and then assign permissions/limits based on those - but even before OIDC is there you can have manual group creation and add users, then set options.

**Comment by fallenbagel** [comment]
> Hello, I'm attempting to move from Ombi and there's a few concerning things that I thought would already be in Seerr that aren't (don't get me wrong - very greatful for this free app!):
> 
> - "Global" Movie/Series Request isn't clear - global to me sounds like cumulatively across all users, everyone can only hit x number of movies/series requests (so if user A and user B request 10 - and the global limit is 20 - they nobody can request anymore). Or is it that it's per-user and a "global default"?

No. Global means the global default applied per user.

> - How can we define more series limits? e.g. The Simpsons has 37 seasons - so if that's counted as 1 series, that's a large risk allowing that through. Can we define number of episodes or seasons for series?

Limits are per season.

> - Is it possible to use group-based limits? E.g. maybe when the OIDC feature is implemented you can pull the in the users groups, and then assign permissions/limits based on those - but even before OIDC is there you can have manual group creation and add users, then set options.

Not yet.

**Reply by j007bond007** [reply]
Thanks very much for the response. That helps clear it up :)
